protos/deployment.proto

```diff
@@ -62,6 +64,13 @@ message DeploymentSpec {
       string function_auth_level = 5;
   }
 
+  message AwsEc2OperatorConfig {
+    string region=1;
+    string min_capacity=2;
+    string max_capacity=3;
+    string desired_capacity=4;
```

Comment:
we should also include other variables that users might want to adjust, such as ami, instance type and etc